### ISSUE TITLE
crash on a QR code

### ISSUE THREAD

**Issue by zapashcanon** [paolostivanin/OTPClient#380]
Hi,

I'm trying to configure a google account to use OTPClient. I made a screenshot of the provided QR code and first tried to import it directly via "Load QR" > "from file", then I tried "Import from third party" > "Load Google migration QR" > "from file".

In both cases, OTPClient crashes with the following:

```shell-session
$ otpclient
corrupted size vs. prev_size while consolidating
[1]    5063 IOT instruction  otpclient
```

Note that I'm not going through Google Authenticator export as I don't have access to this app. I'm getting the QR code from this URL: https://myaccount.google.com/two-step-verification/authenticator

Cheers!

**Comment by paolostivanin** [comment]
hello,
thanks for reporting this. Can you please provide a reproducer? Thanks

**Comment by zapashcanon** [comment]
> can you please provide a reproducer?

I can't really post the QR code because it contains some sensible informations, but you should be able to reproduce by going to the link I gave ?

I'm using version 3.7.0 (on Debian Unstable).

**Comment by paolostivanin** [comment]
Yes, I can do that, but it's always faster with a provided reproducer (which should never contain sensitive information!), especially because I'm not using that service. 

I see that unstable has 4.0.2, please upgrade to that version since it contains a fix for QR parsing (#374)

thx

**Comment by zapashcanon** [comment]
It works on the latest version. Thanks!

**Comment by fvcr** [comment]
> I see that unstable has 4.0.2, please upgrade to that version since it contains a fix for QR parsing (#374)

Hello @paolostivanin and @zapashcanon,

Thank you for noting that we have updated `otpclient` on Debian Unstable.

I recently added `otpclient` to the Stable Backports repository, so users on Debian Stable can now access the latest version by adding the following entry to their `/etc/apt/sources.list.d/bookworm-backports.list`:

```
deb http://deb.debian.org/debian bookworm-backports main
```

Then, run the following commands:

```
sudo apt update
sudo apt install -t bookworm-backports otpclient
```

See:
[Backports Instructions](https://backports.debian.org/Instructions/)


Best regards.

**Comment by paolostivanin** [comment]
cool, thanks a lot @fvcr :smile:
